packages/block-library/src/navigation-menu/style.scss

```diff
@@ -106,4 +90,42 @@
 		}
 	}
 
+	// Menu event activation.
+	&.is-activated-by-hover li {
+		&:hover,
+		&:focus-within {
+			cursor: pointer;
+			z-index: 99999;
+		}
+
+		// Submenu Display
+		&:hover > ul,
+		&:focus-within > ul,
+		& ul:hover,
+		& ul:focus {
+			visibility: visible;
+			opacity: 1;
+			display: block;
+		}
+	}
+
+	&.is-activated-by-click li {
+		input.wp-block-navigation-menu-item__toggle-handler {
+			display: none;
```

Comment:
This will make the control impossible to access for assistive tech. Better to use the `VisuallyHidden` component or a CSS solution to hide visually but retain access for assistive tech:

See also https://wordpress.slack.com/archives/C02QB2JS7/p1573637389298100